apps/web/modules/ee/teams/team-list/components/team-settings/delete-team.tsx

```diff
@@ -38,31 +40,38 @@ export const DeleteTeam = ({ teamId, membershipRole }: DeleteTeamProps) => {
     setIsDeleting(false);
   };
 
-  const { isMember } = getAccessFlags(membershipRole);
+  const DeleteButton = () => (
```

Comment:
Can we simply use this a button, instead of making this a function that returns JSX 🤔